docs: include information on getting started (#103)
newrelic/newrelic-cli: README.md

* Deployment Markers: Easily record an APM Application deployment within
  New Relic.

### Getting Started

> For a quick guide on getting started with this project, see our [Getting

Reviewer: Minor wording suggestion

```suggestion
For a quick guide on getting started with the New Relic CLI, see our [Getting
```

Author: Done.
